tests/page_test.go

```diff
@@ -1633,6 +1633,7 @@ func TestPageIsHidden(t *testing.T) {
 	}
 }
 
+//nolint:tparallel
```

Comment:
Couldn't find a way of running the subtests in parallel. The test server seemed to close before any of the tests ran 🤔

Reply:
`t.Cleanup` helped here.